### DISCUSSION TITLE
Is there any way to install for nvim v0.7.2?

### DISCUSSION THREAD

**Discussion by 1Codealot** [NvChad/NvChad#2466] [Q&A] [answered]
I'm using debian stable and the nvim that I have is 0.7.2 and nvchad says 0.9.0 is needed but there's no way to update it. Is there an older version that I can install?

**Comment by siduck** [comment, marked as answer]
hi @1Codealot no! 0.9 is needed. When I was on debian, I was using this https://github.com/MordechaiHadad/bob to manage neovim versions 

or you can try makedeb + mpr + mist :D
